tools/stress-cluster/scripts/find_all_stress_packages.ps1

```diff
@@ -0,0 +1,53 @@
+param(
```

Comment:
nit: We generally don't use snake casing for PS file names. It is generally kabab or Pascal.